### PR TITLE
[Feat] openyurt fuzz support

### PR DESCRIPTION
**Pull request by huiwq1990**
Signed-off-by: huiwq1990 <huiwq1990@163.com>

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
I'm going to ask the panel if we approve of this.

File: projects/openyurt/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ENV PROJECT_ROOT="${GOPATH:-/root/go}/src/github.com/openyurtio"
+
+ARG REPOSITORY_OWNER="openyurtio"
+ARG REPOSITORIES="yurt-app-manager"
+
+RUN mkdir -p "${PROJECT_ROOT}"
+RUN for repo in ${REPOSITORIES}; do git clone --depth 1 "https://github.com/${REPOSITORY_OWNER}/${repo}" "${PROJECT_ROOT}/${repo}"; done
+RUN git clone --depth 1 https://github.com/${REPOSITORY_OWNER}/cncf-fuzzing
+
+COPY build.sh $SRC/
```
Comment (jonathanmetzman) [inline projects/openyurt/Dockerfile:28]: There's no build.sh here?

Comment (huiwq1990) [reply projects/openyurt/Dockerfile:28]: Sorry, I forget commit it. It's fixed.

**Comment by jonathanmetzman** [comment]
We approve of adding this project. Please fix the build failures and let me know when this is ready

**Comment by huiwq1990** [comment]
> We approve of adding this project. Please fix the build failures and let me know when this is ready

@jonathanmetzman hi, related issues alreay fixed, please review the pr.

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**